tests/test-import.js

```diff
@@ -18,6 +18,24 @@ describe('layout: importer', function () {
   const dirBig = __dirname + '/test-data/dir-big'
   const dirNested = __dirname + '/test-data/dir-nested'
 
+  // check to see if missing empty dirs need to be created
+  fs.exists(__dirname+'/test-data/dir-nested/dir-another', function(exists){
+    if(!exists){
+      fs.mkdir(__dirname+'/test-data/dir-nested/dir-another', function(){
+        fs.exists(__dirname+'/test-data/dir-nested/level-1/level-2', function(exists){
```

Comment:
also note that fs.exists is deprecated:
https://nodejs.org/api/fs.html#fs_fs_exists_path_callback

use `stat` instead and here since there is no need for parallelism, you can even use the sync ones :)